### PR TITLE
Analytics token

### PR DESCRIPTION
**Pull request by tcorbettclark**
See issue #9

### CODE REVIEW

File: encapsia_api/analytics.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+
+def analytics_connect():
+    """Return an EncapsiaApi object connected to this encapsia server.
+
+    Only intended to be used from within Encapsia Analytics.
+
+    Typical usage:
+
+        !pip install encapsia_api
```
Comment (petrem) [inline encapsia_api/analytics.py]: On install from a notebook (or terminal), typically the user will have to restart the jupyter kernel so it "sees" the new package. Consider adding this information as a comment here.

File: pyproject.toml [hunk 2]
```diff
@@ -5,7 +5,7 @@ description = "Client API for talking to an Encapsia system."
 readme = "README.md"
 authors = ["Timothy Corbett-Clark <timothy.corbettclark@gmail.com>"]
 license = "MIT"
-long_description_content_type = "text/markdown"
+# long_description_content_type = "text/markdown"
```
Comment (petrem) [inline pyproject.toml]: please remove

Comment (tcorbettclark) [reply pyproject.toml]: Agreed. I thought I had actually. Are you looking at an old change?

File: encapsia_api/analytics.py (hunk 1 above)
Comment (tcorbettclark) [reply encapsia_api/analytics.py]: Interesting. That's not been my experience. I often install packages using this approach, and have never needed to restart a kernel.

**Pull request merged**